classes/PodsI18n.php

```diff
@@ -196,4 +205,240 @@ private static function default_strings() {
 
 	}
 
+	/**
+	 * Get current language information from Multilingual plugins
+	 *
+	 * @since 2.6.6
+	 * @since 2.7 Moved to this class from PodsAPI
+	 *
+	 * @param array $args (optional) {
+	 *     @type bool $redo Rerun logic?
+	 * }
+	 *
+	 * @return array
+	 */
+	public static function get_current_language( $args = array() ) {
+
+		$args = wp_parse_args( $args, array(
+			'redo' => false,
+		) );
+
+		if ( ! $args['redo'] && null !== self::$current_language_data ) {
```

Comment:
Do we have something already somewhat standardized for cache-busting or semi-related to use as the key name?  We don't seem to use 'redo' anywhere else.   @sc0ttkclark ?